config/environments/development.rb

```diff
@@ -14,8 +14,16 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: 'rubygamedev.dev' }
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.